src/bindings/js/node/lib/addon.ts

```diff
@@ -205,6 +205,13 @@ interface Core {
     deviceName: string,
     properties: { [key: string]: string | number | boolean },
   ): void;
+  /**
+   * It queries the model passed for the requested property.
+   * @param model The passed model to query the property.
+   * @param deviceName The name of a device.
+   * @param properties An object with the property name - property value pairs.
+   */
+  queryModel(model: Model, deviceName: string, properties?: {[key: string]: string | number | boolean}): {[key: string]: string | number | boolean};
```

Comment:
```suggestion
  queryModel(
    model: Model, 
    deviceName: string, 
    properties?: {[key: string]: string | number | boolean},
  ): { [key: string]: string | number | boolean };
```
This line is too long. It should be <80. Can you use `eslint` on your machine?

Reply:
Done